test/test_options.py

```diff
@@ -42,14 +43,53 @@ def test_version():
     version = test_utils.execute_command(host_bin, control_protocol, test_dir, "-v")
     changelog = Path(__file__).parents[1] / "CHANGELOG.rst"
     assert changelog.is_file(), f"Could not find {changelog}"
-
-    with open(changelog, 'rt') as f:
-
-        lines = f.readlines()
-        version_line = 3
-
-        print(lines[version_line])
-        assert str(version[0]) == lines[version_line].strip()
+    assert str(version[0]) == open(changelog, 'rt').readlines()[3].strip()
+
+if platform.uname()[1] == "raspberrypi":
+    def test_dfu_app():
+        test_dir, _, control_protocol, _ = test_utils.get_dummy_files()
+
+        changelog = Path(__file__).parents[1] / "CHANGELOG.rst"
+        assert changelog.is_file(), f"Could not find {changelog}"
```

Comment:
we should already know that changelog exists and have a path to it from lines 43-44

Reply:
That is a test for a different host app, so I preferred to keep the check.